pkg/install/v1alpha1/openebs_crds.go

```diff
@@ -134,6 +134,30 @@ spec:
     # shortNames allow shorter string to match your resource on the CLI
     shortNames:
     - csp
+  additionalPrinterColumns:
```

Comment:
What is the minimum version required to have this working?

Reply:
k8s 1.11 added this in commit message.